src/Dapper.Bulk.Shared/PropertiesCache.cs

```diff
@@ -102,7 +102,8 @@ public static List<PropertyInfo> ComputedPropertiesCache(Type type)
             foreach (var prop in props)
             {
                 var columnAttr = prop.GetCustomAttributes(false).SingleOrDefault(attr => attr.GetType().Name == "ColumnAttribute") as dynamic;
-                ret.Add(prop.Name, columnAttr != null ? (string)columnAttr.Name : prop.Name);
+                // if the column attribute exists, and specifies a column name, use that, otherwise fall back to the property name as the column name
+                ret.Add(prop.Name, columnAttr != null ? (string)columnAttr.Name??prop.Name : prop.Name);
```

Comment:
consider this syntax pls, that would be more obvious
`((string)columnAttr?.Name) ?? prop.Name`